src/ai_hawk/llm/llm_manager.py

```diff
@@ -594,7 +594,7 @@ def answer_question_from_options(self, question: str, options: list[str]) -> str
         prompt = ChatPromptTemplate.from_template(func_template)
         chain = prompt | self.llm_cheap | StrOutputParser()
         output_str = chain.invoke(
-            {"resume": self.resume, "question": question, "options": options})
+            {"resume": self.resume, "job_application_profile": self.job_application_profile, "question": question, "options": options})
```

Comment:
can you move all strings used to constants.py and use them here.

Reply:
Also more selection prompt to prompts.py file. line 438

Reply:
I've moved prompts and strings to respective files.